bindings/ocaml/test_bindings.ml

```diff
@@ -28,6 +30,7 @@ let _ =
   test_ks T.KS_ARCH_X86 T.KS_MODE_64 "add rax, rcx";
   test_ks T.KS_ARCH_X86 T.KS_MODE_32 ~syntax:T.KS_OPT_SYNTAX_ATT "add %ecx, %eax";
   test_ks T.KS_ARCH_X86 T.KS_MODE_64 ~syntax:T.KS_OPT_SYNTAX_ATT "add %rcx, %rax";
+  test_ks T.KS_ARCH_X86 T.KS_MODE_32 ~syntax:T.KS_OPT_SYNTAX_RADIX16 "add eax, 0x15";
```

Comment:
the option RADIX16 requests the engine to treat all the immediates without `0x` prefix as hexadecimal. so in the input above, you actually want to compile "add eax, 15", rather than "add eax, 0x15"?